data/sql/updates/pending_db_world/rev_1559491515154304500.sql

```diff
@@ -0,0 +1,42 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1559491515154304500');
+
+SET @SWIFTSPEAR := 30395;
+SET @GOSSIP :=9906;
+
+DELETE FROM `gossip_menu_option` WHERE `menuid`=@GOSSIP AND `optionid`=0;
```

Comment:
Please remove the "gossip_menu_option" part, this already exists in the DB, and is better, because it has it's broadcast text ID set.